edge_computing/image_base_install/ibi_deploying_sno_clusters/ibi-edge-image-based-install.adoc

```diff
@@ -0,0 +1,62 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ibi-edge-image-based-install"]
+= About image-based deployments for managed {sno}
+include::_attributes/common-attributes.adoc[]
+:context: ibi-edge-image-based-install
+
+toc::[]
+
+When a host preinstalled with {sno} using an image-based installation arrives at a remote site, a technician can easily reconfigure and deploy the host in a matter of minutes.
+
+For clusters with a hub-and-spoke architecture, to complete the deployment of a preinstalled host, you must first define site-specific configuration resources on the hub cluster for each host. These resources contain configuration information such as the properties of the bare-metal host, authentication details, and other deployment and networking information. 
+
+The Image Based Install (IBI) Operator creates a configuration ISO from these resources, and then boots the host with the configuration ISO attached. The host mounts the configuration ISO and runs the reconfiguration process. When the reconfiguration completes, the {sno} cluster is ready.
+
+[NOTE]
+====
+You must create distinct configuration resources for each bare-metal host. 
+====
+
+See the following high-level steps to deploy a preinstalled host in a cluster with a hub-and-spoke architecture:
+
+* Install the IBI Operator on the hub cluster.
+* Create site-specific configuration resources in the hub cluster for each host.
+* Apply these resources in the hub cluster. 
```

Comment:
Not clear to me how do we create resources in the previous step without applying them

Reply:
I think this phrasing comes from the use of both `oc create` vs `oc apply` commands. If you skim through [this section](https://79610--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_base_install/ibi_deploying_sno_clusters/ibi-edge-image-based-install#create-config-iso_ibi-edge-image-based-install), some steps use `oc create` while others use `oc apply`.

If I understand it correctly, both commands do the same (creating resources) but differently? `oc create` can only create resources that do not exist yet, while `oc apply` can handle updating/replacing existing resources.

If that is right, would it make sense to use `oc create` throughout the procedure and update the bullet point as follows?

- Install the IBI Operator on the hub cluster.
- Create site-specific configuration resources in the hub cluster for each host.
- The IBI Operator creates a configuration ISO from these resources and boots the target host with the configuration ISO attached.
- The host mounts the configuration ISO and runs the reconfiguration process. When the reconfiguration completes, the single-node OpenShift cluster is ready.

Reply:
The following is from the DO280 training:

`The kubectl apply command can also create resources with the same -f option that is illustrated with the kubectl create command. However, the kubectl apply command can also update a resource.

Updating resources is more complex than creating resources. The kubectl apply command implements several techniques to apply the updates without causing issues.

The kubectl apply command writes the contents of the configuration file to the kubectl.kubernetes.io/last-applied-configuration annotation. The kubectl create command can also generate this annotation by using the --save-config option. The kubectl apply command uses the last-applied-configuration annotation to identify fields that are removed from the configuration file and that must be cleared from the live configuration.

Although the kubectl create -f command can create resources from a manifest, the command is imperative and thus does not account for the current state of a live resource. Executing kubectl create -f against a manifest for a live resource gives an error. In contrast, the kubectl apply -f command is declarative, and considers the difference between the current resource state in the cluster and the intended resource state that is expressed in the manifest.

For example, to update the container's image from version v1.0 to latest, first update the YAML resource manifest to specify the new tag on the image. Then, use the kubectl apply command to instruct Kubernetes to create a version of the deployment resource by using the updated image version that is specified in the manifest.`


Re "would it make sense to use oc create throughout the procedure"  - not sure if we have some resources already existing - need to look separately in each case.

Re "update the bullet point as follows" - LGTM

Reply:
Thank you! 
> not sure if we have some resources already existing - need to look separately in each case.

In this case, I won't change the commands for now, unless you can confirm that certain instances in the docs are incorrect.